CorsixTH/Src/th_lua_ui.cpp

```diff
@@ -25,9 +25,9 @@ SOFTWARE.
 #include "th_map.h"
 #include <algorithm>
 
-struct THWindowBase_t {};
+class abstract_window {};
```

Comment:
not sure about `abstract_window`, sounds more like something that deals with rectangular areas of some size with generic operations on it, rather than a base class that provides common functionality for a gui, but ymmv.

Reply:
either way it's short for "abstract base class", I like abstract better these days, so I'm keeping it :p